joomla/joomla-cms
# [4.0] Multi category - blog layout view

Pull Request for Issue #24383 .

### Summary of Changes

Changing this 2 files makes possible to display articles from choose categories in the same view, page.

### Testing Instructions

Create a new menu type feature articles, choose more than one categories  after article order parameter there is a new parameter named Featured Articles (Yes- show only features articles, No- show also non featured articles).
Making an patch with this PR should allow testing this feature, I have not tested this.

### Expected result

In the render menu there should be just featured articles if parameter is set to yes, there should be also non featured articles if parameter is set to no.

### Actual result



### Documentation Changes Required
Now feature menu allows to display items that are not featured. A mutlicategory view (with optional featured filter)
